### PR TITLE
Fix jballerina unit tests

### PR DESCRIPTION
**Pull request by azinneera**
- org.ballerinalang.test.balo.constant.SimpleConstantBaloNegativeTest
- org.ballerinalang.test.balo.constant.MapConstantPanicInBaloTest
- io.ballerina.semantic.api.test.SymbolBIRTest
- org.ballerinalang.test.balo.types.NeverTypeBaloTest

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java [hunk 1]
```diff
@@ -33,8 +36,11 @@
 
     @BeforeClass
     public void setup() {
-//        BCompileUtil.compile("test-src/balo/test_projects/test_project", "testorg", "foo");
-//        BCompileUtil.compile("test-src/balo/test_projects/test_project", "testorg", "records");
+        CompileResult compileResult = BCompileUtil.compileAndCacheBalo("test-src/balo/test_projects/test_project");
+        if (compileResult.getErrorCount() != 0) {
```
Comment (NipunaRanasinghe) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java:40]:
```suggestion
        if (compileResult.getErrorCount() > 0) {
```

File: tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project/Ballerina.toml [hunk 2]
```diff
@@ -2,6 +2,6 @@
 org= "testorg"
 name="foo"
 version= "1.0.0"
-exported=["testpkg.foo"]
+exported=["foo", "foo.records"]
 
 [dependencies]
```
Comment (pramodya1994) [inline tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project/Ballerina.toml:7]: Shall we add new line?

File: tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project_negative/Ballerina.toml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-[project]
-# Name of the package
-org-name = "testorg"
+[package]
+org = "testorg"
+name = "foo"
 version = "1.0.0"
```
Comment (pramodya1994) [inline tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project_negative/Ballerina.toml:4]: Shall we add a new line

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java [hunk 4]
```diff
@@ -33,8 +36,11 @@
...
+            Arrays.stream(compileResult.getDiagnostics()).forEach(System.out::println);
+            Assert.fail("Compilation contains error");
```
Comment (NipunaRanasinghe) [inline tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java:42]:
```suggestion
            Assert.fail("Compilation contains errors");
```

**Review by NipunaRanasinghe — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java (hunk 4 above)
Comment (azinneera) [reply tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java]: Fixed in 9c62bc9

File: tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java (hunk 1 above)
Comment (azinneera) [reply tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/types/NeverTypeBaloTest.java]: Fixed in 9c62bc9

File: tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project_negative/Ballerina.toml (hunk 3 above)
Comment (azinneera) [reply tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project_negative/Ballerina.toml]: Fixed in 9c62bc9

File: tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project/Ballerina.toml (hunk 2 above)
Comment (azinneera) [reply tests/jballerina-unit-test/src/test/resources/test-src/balo/test_projects/test_project/Ballerina.toml]: Fixed in 9c62bc9

**Review by pramodya1994 — APPROVED** [review]

**Pull request merged**